flavours/form/manifest.fragment.json

```diff
@@ -4,9 +4,10 @@
   "fields": {
     "username": {
       "type": "text",
-      "label": "login"
+      "label": "login",
+      "identifier": true
```

Comment:
Why is `identifier` necessary here?

Reply:
For username to be detected as the identifier field, which is important in Harvest.

Eventually since I do not understand this feature and since it seems not to be supported by the vault, I simplified and use `login`.